content/en/users/tutorials/htc-job-submission/_index.md

```diff
@@ -0,0 +1,697 @@
+more supported middleware.
+
+In this section we will document querying the EGI Information System to retrieve
+information about the available resources.
+
+> Those examples are relying on the Top BDII maintained by EGI Foundation:
+> `ldap://lcg-bdii.egi.eu:2170`
+
+Information about resources is documented according to the
+[GLUE Schema](https://gridinfo-documentation.readthedocs.io/en/latest/glue.html).
+
+Information in the Top BDII is provided for two GLUE schema versions:
+
+- [GLUE 1.3](https://redmine.ogf.org/dmsf_files/61): the legacy version of the
+  specification, under the LDAP base `Mds-Vo-Name=local,o=grid`.
+- [GLUE 2.0](https://www.ogf.org/documents/GFD.147.pdf): the most recent version
+  of the specification, under the LDAP base `GLUE2GroupID=grid,o=glue`.
+
+> Some resources may not yet be exposed via GLUE 2.0, and it may be required to
+> use GLUE 1.3 for those ones.
+
+```shell
+# Dumping all the information from GLUE 2.0
+$ ldapsearch -x -H ldap://lcg-bdii.egi.eu:2170 -b "GLUE2GroupID=grid,o=glue"
+```
+
+The following queries can be used to retrieve information about **all** the
+Computing Elements of a given type. You will likely be able to use only a subset
+of them, only the ones supporting the Virtual Organisation you are a member of,
+and for which you have a valid VOMS proxy.
+
+### Querying for HTCondorCE compute resources
+
+Most, if not all the HTCondorCE should be discoverable via GLUE 2.0.
+
+```shell
+# Querying for all HTCondorCE compute resources, using GLUE 2.0
+$ ldapsearch -x -LLL -H ldap://lcg-bdii.egi.eu:2170 \
+    -b "GLUE2GroupID=grid,o=glue" \
+    '(&(objectClass=GLUE2Endpoint)(GLUE2EndpointImplementationName=HTCondor))' \
+    GLUE2EndpointInterfaceName \
+    GLUE2EndpointImplementationVersion \
+    GLUE2EndpointURL
+
+# To have the version of the HTCondor batch system
+$ ldapsearch -x -LLL -H ldap://lcg-bdii.egi.eu:2170 \
+    -b "GLUE2GroupID=grid,o=glue" \
+   '(&(objectClass=GLUE2Manager)(GLUE2ManagerProductName=HTCondor))' \
+   GLUE2ManagerProductVersion \
+   GLUE2ComputingManagerTotalLogicalCPUs \
+   GLUE2ComputingManagerComputingServiceForeignKey
+```
+
+It's also possible to look into GLUE 1.3.
+
+```shell
+# Querying for all HTCondorCE compute resources, using GLUE 1.3
+$ ldapsearch -x -LLL -H ldap://lcg-bdii.egi.eu:2170 \
+    -b "Mds-Vo-Name=local,o=grid" \
+    '(&(objectClass=GlueCE)(GlueCEInfoJobManager=HTCondorCE))' \
+    GlueCEImplementationVersion GlueCEImplementationName \
+    GlueCEStateStatus GlueCEUniqueID GlueServiceEndpoint GlueServiceType
+```
+
+### Querying for ARC-CE compute resources
+
+Most, if not all the ARC-CE should be discoverable via GLUE 2.0.
+
+```shell
+# Querying for all HTCondorCE compute resources, using GLUE 2.0
+$ ldapsearch -x -LLL -H ldap://lcg-bdii.egi.eu:2170 \
+    -b "GLUE2GroupID=grid,o=glue" \
+    '(&(objectClass=GLUE2Endpoint)(GLUE2EndpointImplementationName=ARC-CE))' \
```

Comment:
i think the correct value for GLUE2EndpointImplementationName is _nordugrid-arc_

Reply:
yes, it should be GLUE2EndpointImplementationName=nordugrid-arc

Reply:
Fixed, thanks!